integration/spark-common-test/src/test/resources/complexdatawithheader.csv

```diff
@@ -0,0 +1,101 @@
+  deviceInformationId,channelsId,ROMSize,purchasedate,mobile,MAC,locationinfo,proddate,gamePointId,contractNumber
```

Comment:
Is it required to add 100 rows? Can it be less rows for testcase to work?

Reply:
@jackylk yes we can reduce no of rows. Done.